test/old_mimic_model2.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Same comment as I had last time - let's only store the parts we absolutely need for this testing

Reply:
that was already done - I cut out the non-mimic explainer parameters from the file